app/javascript/packs/thankYou/customJavascripts/validators/registrationValidator.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener('DOMContentLoaded', () => {
+  $('#register_form').form({
+    fields: {
+      email: {
+        identifier  : 'user[email]',
+        rules: [
+          {
+            type   : 'email',
+            prompt : 'Please enter a valid Email.'
+          }
+        ]
+      },
+      password: {
+        identifier  : 'user[password]',
+        rules: [
+          {
+            type   : 'empty',
+            prompt : 'Please enter a Password.'
+          },
+          {
+            type   : 'minLength[6]',
+            prompt : 'Password should be minimum 6 characters long.'
+          }
+        ]
+      },
+      password_confirmation: {
+        identifier  : 'user[password_confirmation]',
+        rules: [
+          {
+            type   : 'empty',
+            prompt : 'Please enter a Password Confirmation.'
+          },
+          {
+            type   : `match[user[password]]`,
+            prompt : 'Password Confirmation doesn\'t match Password.'
```

Comment:
https://github.com/fnando/i18n-js

Reply:
it will be in a new issue

Reply:
Doesn't this gem works with sprockets only ?